lib/conjur/variable.rb

```diff
@@ -200,9 +200,13 @@ def version_count
     #
     # @param [Integer] version the **1 based** version.
     # @return [String] the value of the variable
-    def value(version = nil)
+    def value(version = nil, show_expired = false)
```

Comment:
Ok, makes sense to make it a hash. I also realized I forgot to document it, I'll add that, too.